### ISSUE TITLE
scrape character URLs from wikia

### ISSUE THREAD

**Issue by theDavidBarton** [theDavidBarton/the-harry-potter-database#6]
https://harrypotter.fandom.com/wiki/Category:Wizards

```
const count = $$('.category-page__member-link').length
for (let i = 0; i < count; i++) {
  console.log($$('.category-page__member-link')[i].href)
}
```

**Comment by theDavidBarton** [comment]
https://harrypotter.fandom.com/wiki/Hannah_Abbott

```
const count = $('aside > section > div').length
for (let i = 0; i < count; i++) {
  console.log($('aside > section > div')[i].innerText)
}
```

**Comment by theDavidBarton** [comment]
closed by #9
